src/js/tab.js

```diff
@@ -945,6 +945,12 @@ vAPI.tabs.registerListeners();
             }
         }
 
+        window.store.dispatch(window.actions.tabs.setBlockerCount({
```

Comment:
so originally we didn't have a good domain for this, now that we do,  think this would be better

```
actions.tabs.assign({ [tabId]: otherstuff  })
```

Reply:
or wait.. i guess that's deeply nested..

Reply:
how about.. we add an action thats like  `assignIn`

```
actions.tabs.assignIn(tabId, {
  type: 'ads',
  count: pageStore.perLoadBlockedRequestCount 
})
```

Reply:
I think `assignIn` is fine.  However I'd omit having 2 arguments, keep it to 1.

Reply:
Hmm I don't see how you could have one argument for this without adding a key collision. wha would you call the key? `key`? what if the target object already has a `key` key?